src/lib.rs

```diff
@@ -1133,6 +1133,12 @@ impl Element {
         self.find(Locator::Css(&locator)).await?.click().await
     }
 
+    /// Find and click an `<option>` element by its visible text/label.
+    pub async fn select_by_label(mut self, label: &str) -> Result<Client, error::CmdError> {
+        let locator = format!(r".//option[normalize-space(.)='{}']", label);
```

Comment:
What does the `normalize-space` do here?
As for quoting, that's a tricky one. I think trying to do quoting ourselves is just going to lead to problems, so I'd probably mention in the documentation that it's up to the caller to escape `'` characters :)

Reply:
As I can state `normalize-space` was used to handle invisible from IDEAs perspective sings like `&amp` etc.

Overall looking for something by the label is a tricky one on its own. So I think you're right that we should put the responsibility for the proper lookup string on the caller.

I'll update the PR in the nearest (I hope)